File: Ejercicios programación/Ejercicios python/1 py/1,1.py
num1 = float(input("Ingrese el primer número: "))


num2 = float(input("Ingrese el segundo número: "))


operacion = input("Ingrese la operación (suma, resta, multiplicación, división): ")


if operacion == "suma":
    resultado = num1 + num2
    print("El resultado es:", resultado)
elif operacion == "resta":
    resultado = num1 - num2
    print("El resultado es:", resultado)
elif operacion == "multiplicación":
    resultado = num1 * num2
    print("El resultado es:", resultado)
elif operacion == "división":
    if num2 == 0:
        print("Error: No se puede dividir entre cero.")
    else:
        resultado = num1 / num2
        print("El resultado es:", resultado)
else:
    print("Operación no válida. Intente de nuevo.")

File: Ejercicios programación/Ejercicios php/1 php/1,1.php
<?php
echo "Ingrese el primer número: ";
$numero1 = trim(fgets(STDIN));


echo "Ingrese el segundo número: ";
$numero2 = trim(fgets(STDIN));
echo "Seleccione una operación: \n";
echo "1. Suma\n";
echo "2. Resta\n";
echo "3. Multiplicación\n";
echo "4. División\n";
$opcion = trim(fgets(STDIN));
if ($opcion == 1) {
    echo "Resultado: " . ($numero1 + $numero2) . "\n";
} elseif ($opcion == 2) {
    echo "Resultado: " . ($numero1 - $numero2) . "\n";
} elseif ($opcion == 3) {
    echo "Resultado: " . ($numero1 * $numero2) . "\n";
} elseif ($opcion == 4) {
    if ($numero2 != 0) {
        echo "Resultado: " . ($numero1 / $numero2) . "\n";
    } else {
        echo "Error: División por cero no permitida.\n";
    }
} else {
    echo "Opción inválida.\n";
}
?>
